### PR TITLE
Docs: Add notice about mod_evasive to Apache proxy guide

### PR DESCRIPTION
**Pull request by joelhans**
##### Summary

Issue #7568 resurfaced an issue that Netdata's dashboard has with Apache's `mod_evasive` module, which protects from DDoS attacks. Because accessing the dashboard creates a high volume of requests, the module's default config gets the user blocked pretty quickly.

This PR adds text to the Apache proxy guide to give notice about this module and tell users to edit their `mod_evasive.conf` file to mitigate the issue.

##### Component Name

docs

### CODE REVIEW

**Comment by joelhans** [comment]
@kkoomen Feel free to leave feedback too. I can't add you as a reviewer, but perhaps you can create a review yourself. I thought about adding something about configuring just one virtualhost instead of the entire module, but I'm worried it gets too complex at that point. If you have a simple config/explanation that works, we can maybe add that too.

File: docs/Running-behind-apache.md [hunk 1]
```diff
@@ -227,6 +227,35 @@ If you want to enable CSP within your Apache, you should consider some special r
...
+exceed that threshold, and `mod_evasive` will add your IP address to a blocklist.
+
+Our users have found success by setting `DOSPageCount` to `30`. Try this, and raise the value if you continue to see 403
+errors while accessing the dashboard.
+
+```conf
+DOSPageCount 30
+```
+
+Restart Apache with `sudo service apache2 restart`, or the appropriate method to restart services on your system, to
```
Comment (cosmix) [inline docs/Running-behind-apache.md]: I believe there’s no need to restart. They can just `reload`

Comment (kkoomen) [reply docs/Running-behind-apache.md]: @cosmix Just tested it out. A restart is required.

**Comment by kkoomen** [comment]
@joelhans I've added additional documentation and fixed 2 typos you made as well. See #7582.

**Pull request merged**